server/src/common/resolveMoniker.proposed.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
You also need to register the feature as a proposed feature. Otherwise the code will not be available when running the server. See https://github.com/microsoft/vscode-languageserver-node/blob/66f91b3583a19ae621214d908ce19c9b2d182626/server/src/common/api.ts as an example.